docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -79,26 +79,26 @@ H  2.683149  3.028701  3.501215
 
 ### How many images do I need?
 
-It depends... It's usually easier to start with a small number of images and increase if you need to. 
+It depends... It's usually easier to start with a small number of images and increase if you need to. *[A bit vague.]*
 
-Using many (i.e. > 15) will result in slow convergence, but may lead to a more accurate minimum energy pathway. 
+Using many (i.e. > 15) will result in slow convergence, but may lead to a more accurate minimum energy pathway. *[Presumably more images will never give a worse result? Would it be better expressed in terms of a rule of thumb e.g. 10 x the number of expected transition states?]*
 
-Using too few (i.e. < 5) might fail to find the minimum energy pathway. 
+Using too few (i.e. < 5) might fail to find the minimum energy pathway. *[Not clear to me what this means. Partly because I don't have a clear idea of what the "spacing" between images means.]*
```

Comment:
Maybe this recommendation should be changed anyway -- in principle even one image can be sufficient in the case of a very simple path. In the case of a complex pathway, though (e.g. >1 TS), you'll need more.

Reply:
Yes, this was confusing me. It wasn't obvious how to think about the number of images. Can it be viewed in terms of "interpolated" between fixed points start / intermediate / end? The main documentation says it is important to actively choose N (rather than accept the default, 20?), but I didn't know what to with this instruction. You could explain how you chose N for ammonia (easier in terms of a concrete example).